airbyte-integrations/connectors/destination-redshift/build.gradle

```diff
@@ -24,7 +24,7 @@ compileKotlin {
 airbyteJavaConnector.addCdkDependencies()
 
 application {
-    mainClass = 'io.airbyte.integrations.destination.redshift.RedshiftDestination'
+    mainClass = 'io.airbyte.integrations.destination.redshift.RedshiftStagingS3Destination'
```

Comment:
Probably rename it to just `RedshiftDestination` ?

Reply:
done